src/util/index.ts

```diff
@@ -65,12 +66,17 @@ export {
   sendObjectToPlane,
 } from './misc/planeChange';
 export * as string from './lang_string';
+export type {
+  EnlivenObjectOptions,
+  LoadImageOptions,
+} from './misc/objectEnlive';
```

Comment:
here is where i start to have some doubts.
How do i access those types then? Are they on the main export root or do i need and grab them from inside util?